### PR TITLE
Adding back button in settings activity

### LINKED ISSUE #367: Add back button to settings activity
#### Software and hardware versions 
Android v9, device used - Realme X2 pro

#### Problem description
There is no button to go back to home screen from settings screen.
#### Steps to reproduce the problem
It is not a problem but an unavailabilty of feature. TO reproduce, open the app, click on 3 verical dots in top right corner, go to settings. You will find there is no button for backward navigation.
#### Expected behavior
After solving the issue, a button will be added to settings activity.
#### Other information 
I would like to resolve this issue.

### PR DESCRIPTION
**Pull request by htg2704**
Closes #367 

#### What has been done to verify that this works as intended?
Checked on physical device

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Creates better user experience
#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkCode` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

File: skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java [hunk 1]
```diff
@@ -48,21 +54,33 @@
...
         ViewGroup root = getRootView();
         Toolbar toolbar = (Toolbar) View.inflate(this, R.layout.toolbar, null);
         toolbar.setTitle(getString(R.string.settings));
         root.addView(toolbar, 0);
+        toolbar.setNavigationIcon(R.drawable.ic_back_navigation);
+        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                try {
+                    Intent intent = new Intent(SettingsActivity.this, MainActivity.class);
```
Comment (Chromicle) [inline skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java]: You are giving intent here, it is not actually the good practice because if the user click back once again it will come to `settingsActivity` 
please try this 
```
        toolbar.setNavigationIcon(getResources().getDrawable(R.drawable.back_arrow));
        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });
```

File: skunkworks_crow/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -56,7 +64,13 @@
         </activity>
         <activity
             android:name=".views.ui.hotspot.HpSenderActivity"
-            android:configChanges="orientation|screenSize" />
+            android:configChanges="orientation|screenSize"
+            android:parentActivityName=".views.ui.send.SendFormsActivity"
+            >
+            <meta-data
```
Comment (Chromicle) [inline skunkworks_crow/src/main/AndroidManifest.xml]: No need to add metadata for this feat

Comment (htg2704) [reply skunkworks_crow/src/main/AndroidManifest.xml]: Did the needful.

Comment (Chromicle) [reply skunkworks_crow/src/main/AndroidManifest.xml]: I can understand that, can you rephrase that

Comment (htg2704) [reply skunkworks_crow/src/main/AndroidManifest.xml]: Removed configChanges for now. It wasn,t needed for this commit.

Comment (Chromicle) [reply skunkworks_crow/src/main/AndroidManifest.xml]: I still can see in the file changes can you please check that

Comment (htg2704) [reply skunkworks_crow/src/main/AndroidManifest.xml]: it's not there for me, shall I send a new pr?

File: skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java [hunk 3]
```diff
@@ -48,21 +54,28 @@
...
+
+        });
 
         addPreferencesFromResource(R.xml.preferences_menu);
         addPreferences();
     }
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        if (item.getItemId() == android.R.id.home) {
```
Comment (Chromicle) [inline skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java]: why did you removed this

File: skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java [hunk 4]
```diff
@@ -244,6 +258,5 @@ public void afterTextChanged(Editable s) {
         alertDialog.setCancelable(true);
         alertDialog.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
     }
-
```
Comment (Chromicle) [inline skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java]: avoid blank spaces

File: skunkworks_crow/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -45,7 +46,11 @@
                 android:value=".views.ui.main.MainActivity" />
         </activity>
         <activity android:name=".views.ui.instance.InstancesList" />
-        <activity android:name=".views.ui.settings.SettingsActivity" />
+        <activity android:name=".views.ui.settings.SettingsActivity"
```
Comment (Chromicle) [inline skunkworks_crow/src/main/AndroidManifest.xml]: revert whatever changes you did in manifest file, as there is no need

File: skunkworks_crow/src/main/AndroidManifest.xml (hunk 2 above)
Comment (Chromicle) [reply skunkworks_crow/src/main/AndroidManifest.xml]: No need to do new PR you still can work on this

Comment (Chromicle) [reply skunkworks_crow/src/main/AndroidManifest.xml]: please run `./gradlew checkcode` locally
